addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1831,12 +1837,11 @@ export class OdooEditor extends EventTarget {
             }
         }
 
-        const selection = document.getSelection();
-        if (!selection.isCollapsed || !selection.rangeCount) return;
-
-        const block = closestElement(selection.anchorNode, 'P, DIV');
+        const block = this.options.getPowerboxElement();
+        if (block) {
         this._makeHint(block, 'Type "/" for commands', true);
     }
+    }
```

Comment:
Hard to see like this on Github but isn't there an indentation problem here?

Reply:
Yes, fixed.

Reply:
Indeed, at that commit, the indentation was wrong. Fixed.